plugins/woocommerce-blocks/docs/internal-developers/testing/releases/1160.md

```diff
@@ -211,12 +211,12 @@ Here is a [quick demo](https://github.com/woocommerce/woocommerce-blocks/assets/
 
 #### Default template
 
-1. Go to Editor and edit "Single Product" template (Producto individual)
+1. Go to Editor and edit "Single Product" template (Product individual)
 2. Check the Related Products heading is translated and has top and bottom margin
 
 #### Transform from classic template
 
-1. Go to Editor and edit "Single Product" template (Producto individual)
+1. Go to Editor and edit "Single Product" template (Product individual)
```

Comment:
I think the two changes in this file (plugins/woocommerce-blocks/docs/internal-developers/testing/releases/1160.md) should be reverted, they are not typos but text in Spanish.

Reply:
Thanks @albarin for catching that!
Fixed in:
- [Revert change of "Producto" word](https://github.com/woocommerce/woocommerce/pull/48562/commits/4e3d93d908423a8b0937091a3181020d39617d30)
- [Revert change of "Producto" word](https://github.com/woocommerce/woocommerce/pull/48562/commits/3d460ef411177146671c99ab14690db4e140e4e4)